cddagl/ui.py

```diff
@@ -2208,6 +2351,27 @@ def run(self):
 
                     self.completed.emit()
 
+            class TestingImgThread(QThread):

```

Comment:
This check could be skipped, since the image verification happens during the mounting. But I left it in place for consistency.